x-pack/plugins/observability_solution/apm/scripts/diagnostics_bundle/diagnostics_bundle.ts

```diff
@@ -104,8 +105,8 @@ async function getApmIndices(kbnClientOpts: AxiosRequestConfig) {
 async function getFleetPackageInfo(kbnClientOpts: AxiosRequestConfig) {
   const res = await axios.get('/api/fleet/epm/packages/apm', kbnClientOpts);
   return {
-    version: res.data.response.version,
-    isInstalled: res.data.response.status,
+    version: res.data.item.version,
+    isInstalled: res.data.item.status,
```

Comment:
>[!IMPORTANT]
> I'm not sure if this script is still used, but `res.data` doesn't have a `response` object.

To test it, run: `node x-pack/plugins/observability_solution/apm/scripts/create_diagnostics_bundle.js --local`